state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -537,6 +548,19 @@ pub mod pallet {
 		OptionQuery,
 	>;
 
+	/// Stores the details of an affiliate account against the account id of a broker and the
+	/// derived affiliate id.
+	#[pallet::storage]
+	pub type AffiliateAccountDetails<T: Config> = StorageDoubleMap<
+		_,
+		Identity,
+		T::AccountId,
+		Twox64Concat,
```

Comment:
```suggestion
		Identity,
```

nit: since we use the blake hasher to generate this already, it can be Identity.